src/book.py

```diff
@@ -115,7 +146,12 @@ def clean(self, lemmatize=True, deromanize=False, lemma_pos='v', inplace=False):
                     WNLemma.lemmatize(token, pos='n'), pos='v'), pos='a')
                     for token in tokens]
             cleaned = (" ").join(lemmatized)
-
+        else:
+            # Added this bit just to clean up unnecessary spaces in the text.
+            # The Clean Function thus returns a text with no extra spaces.
+            tokens = word_tokenize(cleaned)
+            cleaned = (" ").join(tokens)
+            
```

Comment:
Technically this will introduce spaces around punctuation that might not otherwise exist. If your goal is to replace any grouping of spaces with a single space, I propose this instead:
```suggestion
        else:
            # Added this bit just to clean up unnecessary spaces in the text.
            # The Clean Function thus returns a text with no extra spaces.
            cleaned = re.sub('\s+', ' ', cleaned)
            
```